docs/mobile-apps/features/network-capture.md

```diff
@@ -7,92 +7,182 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+
+In combination with our [Network Throttle](/mobile-apps/features/network-throttling/) functionality to simulate various network scenarios,
+you have the tools to replicate and troubleshoot any performance concerns stemming from network operations.
 
 ## What You'll Need
 
 - A Sauce Labs account ([Log in](https://accounts.saucelabs.com/am/XUI/#login/) or sign up for a [free trial license](https://saucelabs.com/sign-up))
-- A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests. We currently support the following:
-  - Android 6.0 and higher
-  - For Swift: iOS 11.0 and higher
-  - For Objective-C: iOS (all)
+- A native Android, iOS, or iPadOS mobile app that makes HTTP/HTTPS requests.
 
-## Using Network Traffic Capture
+## System-wide Network Capture
 
-To enable network traffic capturing in your tests:
+Our system-wide network capture feature enhances network monitoring by capturing all HTTP/HTTPS requests made from the device, offering
+visibility beyond just a single app or service. It works seamlessly regardless of the app’s framework or if it’s a hybrid application.
+This comprehensive approach provides a holistic view of network activity, giving you deeper insights into how various components of the
+device interact with external services. To ensure the captured data remains relevant and manageable, our system filters out some OS-specific
+services that can otherwise create unnecessary noise.
 
-1. On Sauce Labs, click **Live** > **Mobile App**.
-2. Upload your mobile app to Sauce Labs [through our UI](/mobile-apps/app-storage/#uploading-apps-via-ui) or [our REST API](/mobile-apps/app-storage/#uploading-apps-via-rest-api).
-3. After you’ve uploaded your app, return to the **Live** > **Mobile App** page, hover your mouse over your app, then select **Settings**.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr.png')} alt="Mobile app settings navigation" width="600"/>
-4. Under **Default Settings**, toggle Instrumentation and Network Capture to enable the feature.<br/><img src={useBaseUrl('img/mobile-apps/networkcapturescr2.png')} alt="Mobile app settings navigation" width="800"/>
-5. For Automated Testing only: add the networkCapture capability to your test script. Click the link below corresponding to your framework:
+On iOS/iPadOS devices, this feature works seamlessly without requiring any [resigning](/mobile-apps/automated-testing/ipa-files/#sauce-labs-resigning-enablements)
+or changes to the app, making it simple and efficient to use.
 
-   - [Appium](/dev/test-configuration-options/#saucelabsnetworkcaptureenabled)
-   - [Espresso via saucectl](/mobile-apps/automated-testing/espresso-xcuitest/espresso/#networkcapture)
-   - [XCUITest via saucectl](/mobile-apps/automated-testing/espresso-xcuitest/xcuitest/#networkcapture)
+For Android devices, a minor modification in the manifest is necessary, but all required changes will be handled by us. This means no
```

Comment:
lets call out here that we need to modify the manifest file and call out this to ensure the SSL pinning is disabled, and that our certificate is accepted. 
additionally call out we need the debug version of their app, and make sure that in the secureSDK repackaging prevention is disabled